spec/controllers/users_controller_spec.rb

```diff
@@ -68,15 +78,38 @@
     end
   end
 
-  describe 'DELETE users#destroy' do
-    it 'deletes the user' do
+  describe 'users#destroy' do
+    it 'deletes the current_user account' do
       expect(response).to have_http_status(:ok)
       expect { delete :destroy, params: { id: user.id } }.to change(User, :count).by(-1)
     end
 
-    it 'does not delete any user if the user id is invalid' do
+    it 'returns status code forbidden if the user tries to delete another user' do
+      new_user = create(:user)
+      expect { delete :destroy, params: { id: new_user.id } }.not_to change(User, :count)
+      expect(response).to have_http_status(:forbidden)
+    end
+
+    it 'returns status code forbidden if the user id is invalid' do
```

Comment:
The word "invalid" may be misleading since we use it to indicate invalid params,
Do you find this more informational "returns status code :forbidden for inexistent :id"?

Reply:
It is kind of invalid, no?
It's a string and it's expecting an id.
`params: { id: 'invalid-id' }`

Reply:
Aren't we testing here the case when we're trying to remove a user that does not exist?
This gives the impression that our API is asserting the data type of the inputs our their values considering some as invalid and returning an errored response.